src/SamplesApp/Benchmarks.Shared/Suite/Collections/FreezableListVsImmutableListBench.cs

```diff
@@ -0,0 +1,32 @@
+#if !WINDOWS_UWP
+using BenchmarkDotNet.Attributes;
+using Uno.Collections;
+
+namespace SamplesApp.Benchmarks.Suite;
+
+[MemoryDiagnoser]
```

Comment:
Note: The benchmark output didn't contain anything related to memory :/
Not sure why is that.